src/commands/CommandMusicReplay.java

```diff
@@ -0,0 +1,24 @@
+package commands;
+
+import utilities.abstracts.MusicCommands;
+import utilities.music.MusicManager;
+
+public class CommandMusicReplay extends MusicCommands {
+	@Override
+	public void action(){
+		
+		if(getGuild() == null)
+			return;
+		
+		connectIfNotPlaying();
```

Comment:
There is no handling of the fact that a latest song might not be there (`if(!hasMemory("LATEST_SONG"))`).

There should be some.